db/migrate/20181026182144_add_tasks_table.rb

```diff
@@ -0,0 +1,13 @@
+class AddTasksTable < ActiveRecord::Migration[5.1]
+  def change
+    create_table :tasks, :id => :bigserial do |t|
+      t.string :name
+      t.string :status
```

Comment:
Do we need this and message?

Reply:
We need status to know when the task is completed, we could do without message if we don't want to have any output initially.